src/components/table/table.js

```diff
@@ -919,12 +919,17 @@ export default {
     },
     // Event handlers
     rowClicked(e, item, index) {
+      const getSelection = (document || {}).getSelection || (window || {}).getSelection
```

Comment:
Maybe we should create a util for that. Something like:
```js
const getSelection = () => {
  if (document.getSelection) {
    return document.getSelection()
  }
  if (window.getSelection) {
    return window.getSelection()
  }
  return null
}
```

Reply:
Yeah, thought about that, but not sure if we would be using it anywhere else.

Reply:
Moved to a top of file helper function, as it is used for both `row-clicked` and `head-clicked` events.